interface/src/app/gene/searchMany/gene-search-form.tpl.html

```diff
@@ -1,29 +1,11 @@
-<div class="row">
-  <div class="col-xs-12">
-    <div class="panel panel-default">
-      <div class="panel-heading">
-        <h3>
-          Search for Genes
-        </h3>
-      </div>
-      <div class="panel-body">
-        <div class="col-xs-3">
-          <form class="form">
-            <textarea name="genes" rows="7" ng-model="genesToAdd.query"
-              placeholder="Search genes by symbol or Entrez ID">
-            </textarea>
-            <p class="bg-danger">{{ errors }}</p>
-            <br>
-            <button class="btn btn-primary" ng-click="searchGenes()"
-              ng-disabled="loadingSearchResults">
-              Search <i class="fa fa-search"></i>
-            </button>
-          </form>
-        </div>
-        <div class="col-xs-9">  
-          <search-result-table></search-result-table>
-        </div>
-      </div>
-    </div>
-  </div>
-</div>
+<form class="form">
+  <textarea name="genes" rows="7" ng-model="geneQueries"
```

Comment:
Should this "magic number" 7 be a parameter that can be specified in the `<gene-search-form>` directive (but default to 7)? That way the `gene-search-panel`, which is responsible for layout of this widget, can specify another "size," if needed.